teamB/eungSooKim/app.js

```diff
@@ -53,11 +49,118 @@ app.post("/users", async (req, res, next) => {
             password
         ) VALUES (?, ?, ?);
         `,
-    [name, email, password]
+    [name, email, passWord]
   );
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts/write/:userId", async (req, res) => {
```

Comment:
게시글을 작성하는 api의 endpoint를 호출하는 코드인 것 같네요!

현재 api를 POST method의 `/posts/write/:userId`로 작성해주셨는데 잘 작성해주셨습니다.

다만, 금주에 배우게 될 RESTful API 세션과 관련한 코멘트를 미리 남겨드리자면, REST란 `자원을 이름(자원의 표현)으로 구분하여 해당 자원의 상태(정보)를 주고 받는 모든 것을 의미`합니다. RESTful API는 이름 그대로 REST 기반으로 서비스 API를 구현하는 것을 의미합니다. 

REST하게 API를 개발하고 엔드포인트를 작성할 때에 가장 기본적으로 지향해야 하는 것은 `'자명할 수 있는'` 엔드포인트를 작성하는 것입니다.

여기서 자명하다는 것은 말 그대로 http method + endpoint 만 보고도 해당 엔드포인트로 구현된 API가 어떤 리소스를 목적하는지 알 수 있다는 말로, 조금 더 간단하게 말해 엔드포인트만 보고도 어떤 기능을 하는 API인지 설명돼야 한다는 의미입니다. 

```suggestion
app.post("/posts", async (req, res) => {
```

suggestion으로 작성드린 것과 같이 post method로 http request를 보낸다는 것은 무언가 생성, 혹은 수정한다는 의미이고 게시글을 작성하는 endpoint는 이러한 http method에 더해 `/posts` 만으로 충분히 전달될 것 같습니다. (추가로 REST하게 endpoint를 작성할 때에는 엔드포인트에 write와 같은 동사를 넣지 않습니다. write 또한 무언가를 작성하는 의미이기 때문에 posts와 중복되기 때문이겠죠?)

해당 부분에 대한 자세한 내용은 금주에 있는 RESTful API 세션을 참고해주세요-!